charger/ocpp.go

```diff
@@ -422,11 +456,16 @@ func (c *OCPP) phases1p3p(phases int) error {
 	// NOTE: this will currently _never_ do anything since
 	// loadpoint disabled the charger before switching so
 	// updatePeriod will short-circuit
-	return c.updatePeriod(c.current, c.phases)
+	return c.updatePeriod(c.current)
 }
 
 // // Identify implements the api.Identifier interface
 // Unless charger uses vehicle ID as idTag in authorize.req it is not possible to implement this in ocpp1.6
 // func (c *OCPP) Identify() (string, error) {
 // 	return "", errors.New("not implemented")
 // }
+
+// LoadpointControl implements loadpoint.Controller
+func (c *OCPP) LoadpointControl(lp loadpoint.API) {
+	c.lp = lp
```

Comment:
Hier fehlt noch die Logik Dir die Phasen aus dem LP zu holen und v.a. sie für die Leistungsberechnung auch zu nutzen?

Reply:
Das ist der auskommentierte Teil in den Zeilen (394-399). Wie oben geschrieben, machen die LP-Phasen in allen Fällen, die ich betrachtet habe, keinen Unterschied. 

- c.phases = 0, lpPhases = 1 => phases = 3 (ohne Umschaltung)
- c.phases = 1, lpPhases = 1 => phases = 1 (mit Umschaltung)
- c.phases = 3, lpPhases= 1 => phases = 3 (egal wie) 
- c.phases = 0, lpPhases = 3 => phases = 3 (ohne Umschaltung)
- c.phases = 3, lpPhases= 3 => phases = 3 (egal wie) 

Welchen Fall habe ich nicht betrachtet?

Reply:
Nimm bitte- wie vorgeschlagen- GetPhases. Sonst brauchst du auch den Loadpoint nicht.

Reply:
Ok, für mein Setup macht das keinen Unterschied. Für andere Umgebungen muss man das beobachten.